examples/in-app-messaging-example/README.md

```diff
@@ -0,0 +1,3 @@
+# Knock In App Messaging Example
```

Comment:
```suggestion
# Knock In-App Message Example
```
Nit, but I wonder if we just want to just align our wording to "in-app message" (w/o -ing) everywhere including file/dir names.

Reply:
Yeah, that sounds good to me!